scripts/process_new_format.py

```
@@ -2,10 +2,11 @@
 import logging
 from pathlib import Path
 from pprint import pp
-from typing import List
-from pydantic import BaseModel
-from typing import Optional
+from typing import List, Optional
```

Pylint (3.3.7)

[error] Unable to import 'pydantic'

(E0401)